debugoverlay-ext-netstats/README.md

```diff
@@ -0,0 +1,56 @@
+Network Statistics Extension Module
+===================================
+
+**NetStatsModule** is an extension module for DebugOverlay which shows the network usage.
```

Comment:
could u add more details about the network usage? ex...network usage of wifi, mobile or everything? only the usage of this app or device overall?...etc